### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_ImportFlatFileWizard_SpecifyInputFile_Screenreader : NVDA is incorrectly announcing not required field as required field.

### ISSUE THREAD

**Issue by pranshu512** [microsoft/azuredatastudio#10655]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#MAS1.3.1;#SQL Azure Data Studio;#FTP;#NVDA;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider (system setup)
Commit: 8288360
Date: 2020-05-29T05:54:49.731Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.19041

**Additional Details:**
MAS References: MAS1.3.1
Screenreader : NVDA 2019.3.1

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Connect to server.
3. Install the SQL Server Import extension.
4. Right click on any database under the connected server & select Import Wizard.
5.Navigate to un-required fields in the form.
6.Start Screenreader as NVDA and observe whether NVDA is reading not required field as required field or not.

**Actual:**
NVDA is incorrectly announcing not required field as required field.

**Expected:**
NVDA should not announce un-required field as required field.

**User Impact:**
Screen reader users will get incorrect information about the un-required fields and get confused.

**Attachment link for Reference:**

[10655_A11y_AzureDataStudioStayHealthy_ImportFlatFileWizard_SpecifyInputFile_Screenreader.zip](https://github.com/microsoft/azuredatastudio/files/4707535/10655_A11y_AzureDataStudioStayHealthy_ImportFlatFileWizard_SpecifyInputFile_Screenreader.zip)

**Comment by Rama8345** [comment]
Verified the issue at Azure Data Studio. the issue is not reproducible. So, closing the bug. Please find attachment for reference.
**Github Tags:**#Closed;
![10655_Closed](https://user-images.githubusercontent.com/115216656/221833844-f7e58688-1113-4db7-a422-c76a04135d9c.gif)
